sources/packages/web/src/assets/css/formio-shared.css

```diff
@@ -48,6 +48,10 @@
   line-height: 18px;
 }
 
-.ff-form-container .alert-success{
+.ff-form-container .alert-success {
   display: none;
 }
+
+.align-bullets ul {
+  padding-left: 20px;

```

Comment:
can you share, why this padding for ul @dheepak-aot ?

Reply:
Form.io styles are setting padding:0 at parent level for the container component and because of that <ul> is losing its padding property. 
![image](https://user-images.githubusercontent.com/54600590/134599607-a3d141ee-3cee-475a-bf6b-ff551f7040ca.png)